test/libsolidity/syntaxTests/abiEncoder/v1_accessing_public_state_variable_via_v2_type.sol

```diff
@@ -0,0 +1,17 @@
+struct Item {
+    uint x;
+    uint y;
+}
+
+contract D {
+    Item[][][] public items;
+
+    function test() public view returns (uint) {
+        // The autogenerated getters to not use ABI encoder.
+        Item memory item = this.items(1, 2, 3);
+        return item.x + item.y;
+    }
+}
+// ----
+// TypeError 7364: (202-240): Different number of components on the left hand side (1) than on the right hand side (2).
```

Comment:
How do we make this error go away?

Reply:
Use `(uint a, uint b)` instead of `Item` like in the other test (`v1_accessing_public_state_variable_via_v1_type.sol`). As far as I can see you can't get a struct returned from an automatically generated getter.